android/res/xml/locales_config.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
+    <locale android:name="en" />
+    <locale android:name="ar" />
+    <locale android:name="be" />
+    <locale android:name="bg" />
+    <locale android:name="ca" />
+    <locale android:name="cs" />
+    <locale android:name="da" />
+    <locale android:name="de" />
+    <locale android:name="el" />
+    <locale android:name="en-GB" />
+    <locale android:name="es" />
+    <locale android:name="es-MX" />
+    <locale android:name="eu" />
+    <locale android:name="fa" />
+    <locale android:name="fi" />
+    <locale android:name="fr" />
+    <locale android:name="fr-CA" />
+    <locale android:name="hu" />
+    <locale android:name="in" />
+    <locale android:name="it" />
+    <locale android:name="iw" />
+    <locale android:name="ja" />
+    <locale android:name="ko" />
+    <locale android:name="mr" />
+    <locale android:name="nb" />
+    <locale android:name="nl" />
+    <locale android:name="pl" />
+    <locale android:name="pt" />
+    <locale android:name="pt-BR" />
+    <locale android:name="ro" />
+    <locale android:name="ru" />
+    <locale android:name="sk" />
+    <locale android:name="sv" />
+    <locale android:name="sw" />
+    <locale android:name="th" />
+    <locale android:name="tr" />
+    <locale android:name="uk" />
+    <locale android:name="vi" />
+    <locale android:name="zh" />
+    <locale android:name="zh-HK" />
```

Comment:
Are you sure here? I can see the following names in res/values:

- values-zh-rHK
- values-zh-rMO
- values-zh-rTW

Reply:
add zh-HK or zh-rHK have none difference, because string name file contains `r` is just use tu specifie when you have some strings only for country.
Examples:
If i want translate all strings for Great Britain, i create res file with this name values-en-GB
If i want only translate special strings for Great Britain,  i create res file with this name values-en-rGB. (And strings not translate in this file are take from values-en or values.

I have test with `r` and without, 0 difference.

https://developer.android.com/guide/topics/resources/localization#avoid-creating-more-resource-files-and-text-strings-than-you-need

Reply:
I've read the docs and confirm that the new locale format should work. We use the old one, pre-Android 7 for our resources because we need to support Android 5 and 6.